### DISCUSSION TITLE
Error running WaveTransmittedTurbulence.jl

### DISCUSSION THREAD

**Discussion by Swarnali-D** [CliMA/Oceananigans.jl#3768] [Helpdesk]
I am trying to reproduce the figures of WaveTransmittedTurbulence.jl in julia version 1.9.3
On running `julia --project -e 'using Pkg; Pkg.instantiate()'`, I am met with an error :

`┌ Warning: The active manifest file is an older format with no julia version entry. Dependencies may have been resolved with a different julia version.
└ @ ~/WaveTransmittedTurbulence/Manifest.toml:0
ERROR: Could not locate the source code for the DelimitedFiles package. Are you trying to use a manifest generated by a different version of Julia?
`

How do I overcome this? Or is there any other way I can run the simulations?

**Comment by glwagner** [comment]
Hmm, I think we either need to update the script for julia 1.9.3 or use the same julia version that we used back then for the runs! I think it's probably better to try to update it but let me know what you think.

**Reply by glwagner** [reply]
I jsut realized this is a discussion on Oceananigans.jl. Can we move this to the appropriate repo? I'm assuming you're trying to use this repo: https://github.com/glwagner/WaveTransmittedTurbulence.jl

**Comment by glwagner** [comment]
Please open an issue at https://github.com/glwagner/WaveTransmittedTurbulence.jl

I will delete this after

**Comment by Swarnali-D** [comment]
Sure, thanks!
